src/porepy/numerics/time_step_control.py

```diff
@@ -722,16 +722,15 @@ def is_schedule_in_simulated_times(
         return schedule.size == np.sum(in1d)
 
     # I/O
-    def write_time_information(self, path: Optional[Path] = None) -> None:
```

Comment:
I think this can remain. The "default" case of a None path will trigger no export.

Reply:
Sorry, I somehow scrolled to far below, and just saw the ignoring behavior for the load when path is None. I am OK with your change then!